DfciPkg/Docs/Scenarios/DfciScenarios.md

```diff
@@ -21,9 +21,7 @@ service by an OEM or [Microsoft Cloud Solution Provider](https://docs.microsoft.
 
 ### OEMs that support DFCI
 
-[![Microsoft Surface](collateral/SurfaceLogo_mu.png)](https://docs.microsoft.com/en-us/surface/surface-manage-dfci-guide)
-
-More are in the works...
+[See the latest list at the offical Microsoft documentation site](https://learn.microsoft.com/en-us/autopilot/dfci-management#oems-that-support-dfci)

```

Comment:
It's best to drop the language from the link so it will get replaced based on the user's region.

```suggestion
[See the latest list at the offical Microsoft documentation site](https://learn.microsoft.com/autopilot/dfci-management#oems-that-support-dfci)

```